test-plan.md

```diff
@@ -215,6 +218,52 @@ To setup a test project to verify on you can do:
 * When no changes have been performed on `Pages/Index.cshtml`, there are 0 errors.
 * Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file.
 
+##### Debugging with blazorwasm debug adapter
+
+###### Standalone app
+
+To set up a test project to verify on, create a new Blazor WebAssembly application using the dotnet CLI.
```

Comment:
@JoeRobich the vendors use latest CLI right?

Reply:
I would assume, since they are also doing the .NET 5 preview validation, but I can call it out specifically when I request testing.